src/ErrorBoundary.tsx

```diff
@@ -1,89 +1,49 @@
+import { ErrorBoundary as SentryErrorBoundary } from '@sentry/react';
+import { FallbackRender } from '@sentry/react/dist/errorboundary';
 import * as React from 'react';
 import { ErrorBoundaryContext } from './ErrorBoundaryProvider';
 import { FallbackComponent } from './FallbackComponent';
-import { ErrorBoundaryProps, ErrorBoundaryState } from './types';
+import { ErrorBoundaryProps, FallbackProps } from './types';
 
-export class ErrorBoundary<P extends object = {}> extends React.PureComponent<
-  P & ErrorBoundaryProps<P>,
-  ErrorBoundaryState
-> {
-  public static contextType = ErrorBoundaryContext;
-  public context!: React.ContextType<typeof ErrorBoundaryContext>;
+const wrapFallback = (Component: React.ElementType<FallbackProps>): FallbackRender => {
```

Comment:
this is to mitigate the breaking changes.
In fact there are almost no breaking changes introduced in this whole PR, so the bump should be straightforward.